workloads/web.hbs.md

```diff
@@ -57,3 +57,38 @@ tanzu apps workload apply tanzu-java-web-app --type=web
 
 You can use this to test which applications can function well as serverless web applications,
 and which are more suited to the `server` application style.
+
+## <a id="communication"></a> Communication between `web` workloads
+
+When a workload of the type `web` is created, a Knative service is deployed to the cluster. To access your application,
+you will need the URL for the route created by the Knative Service. You can obtain it by running one of the commands below:
+
+```console
+tanzu apps workload get <WORKLOAD-NAME> --namespace <YOUR-DEVELOPER-NAMESPACE>
+kubectl get ksvc <WORKLOAD-NAME> -n <YOUR-DEVELOPER-NAMESPACE> -ojsonpath="{status.url}"
+```
+
+It is important to note that for communication between workloads within the same Kubernetes namespace, workloads of the type
+`web` should be addressable by referencing both the workload name and the workload's namespace at a minimal. This behaviour is
```

Comment:
I think you mean "minimum" here, but this seems a little indirect.  Maybe "when calling a Knative service, the Service name and namespace are required"?

Reply:
I will rephrase that, thanks!